auron-build.sh

```diff
@@ -29,7 +29,7 @@
 # Description:
 #   Define constants for supported component versions
 # -----------------------------------------------------------------------------
-SUPPORTED_OS_IMAGES=("centos7" "ubuntu24" "rockylinux8" "debian11")
+SUPPORTED_OS_IMAGES=("centos7" "ubuntu24" "rockylinux8" "debian11", "azurelinux3")
```

Comment:
It looks like there’s an extra comma before "azurelinux3" in the array definition. It should be :
```
SUPPORTED_OS_IMAGES=("centos7" "ubuntu24" "rockylinux8" "debian11" "azurelinux3")
```